data_steward/resource_files/schemas/cdm/clinical/survey_conduct.json

```diff
@@ -0,0 +1,146 @@
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "respondent_type_concept_id",
+        "description": "",
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "timing_concept_id",
+        "description": "This is a Concept that represents the timing of the survey. For example this could be the 3-month follow-up appointment.\tThere is no specific domain or class for this field, just choose the one that best represents the value given in the source.",
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "collection_method_concept_id",
+        "description": "This Concept represents how the responses were collected.\tUse the concepts that have the relationship ‘Has Answer’ with the CONCEPT_ID 42529316.",
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "assisted_source_value",
+        "description": "Source value representing whether patient required assistance to complete the survey. Example: ‘Completed without assistance’, ‘Completed with assistance’.",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "respondent_type_source_value",
+        "description": "",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "timing_source_value",
+        "description": "Text string representing the timing of the survey. Example: Baseline, 6-month follow-up.",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "collection_method_source_value",
+        "description": "The collection method as it appears in the source data.",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "survey_source_value",
+        "description": "The survey name as it appears in the source data.",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "survey_source_concept_id",
+        "description": "If unavailable, set to 0.",
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "survey_source_identifier",
+        "description": "Unique identifier for each completed survey in source system.",
+        "type": "string",
+        "mode": "nullable"
+    },
+    {
+        "name": "validated_survey_concept_id",
+        "description": "If unavailable, set to 0.",
+        "type": "integer",
+        "mode": "required"
+    },
+    {
+        "name": "validated_survey_source_value",
+        "description": "Source value representing the validation status of the survey.",
+        "type": "integer",
+        "mode": "nullable"
+    },
+    {
+        "name": "survey_version_number",
+        "description": "Version number of the questionnaire or survey used.",
+        "type": "string",
+        "mode": "nullable"
+    }
+        {
```

Comment:
@MichaelSchmidt82 Codacy and unit tests are failing because of this missing COMMA between these two fields.

Reply:
Thanks, added